README.md

```diff
@@ -1,2 +1,440 @@
+
+Install all required libraries:
+`pip install -r requirements.txt`
+
+## To run the tests on Browserstack [Temporary: For Reviewer purpose only]
+
+1. `robot --variable testType:single --suite e2e .` - Run e2e test suite
+2. `robot --variable testType:single --suite offers .` - Run offers test suite
+3. `robot --variable testType:single --suite user .` - Run user test suite
+4. `robot --variable testType:single --suite login .` - Run login test suite
+5. `robot --variable testType:single --suite product .` - Run product test suite
+6. `robot --variable testType:single --suite offers .` - Run product test suite
+
+## To run the test cases in each test suite parallely on Browserstack [Temporary: For Reviewer purpose only]
+1. `pabot --testlevelsplit --variable testType:single --suite e2e .` - Run e2e test suite
+2. `pabot --testlevelsplit --variable testType:single --suite offers .` - Run offers test suite
+3. `pabot --testlevelsplit --variable testType:single --suite user .` - Run user test suite
+4. `pabot --testlevelsplit --variable testType:single --suite login .` - Run login test suite
+5. `pabot --testlevelsplit --variable testType:single --suite product .` - Run product test suite
+6. `pabot --testlevelsplit --variable testType:single --suite offers .` - Run product test suite
+
+## To run on prem [Temporary: For Reviewer purpose only]
+1. `robot --variable testType:prem --suite e2e .` - Run e2e test suite
+2. `robot --variable testType:prem --suite offers .` - Run offers test suite
+3. `robot --variable testType:prem --suite user .` - Run user test suite
+4. `robot --variable testType:prem --suite login .` - Run login test suite
+5. `robot --variable testType:prem --suite product .` - Run product test suite
+6. `robot --variable testType:prem --suite offers .` - Run product test suite
+
+## To run with local testing [Temporary: For Reviewer purpose only]
+1. `robot --variable testType:local --suite e2e .` - Run e2e test suite
+2. `robot --variable testType:local --suite offers .` - Run offers test suite
+3. `robot --variable testType:local --suite user .` - Run user test suite
+4. `robot --variable testType:local --suite login .` - Run login test suite
+5. `robot --variable testType:local --suite product .` - Run product test suite
+6. `robot --variable testType:local --suite offers .` - Run product test suite
+
+## To run on Docker [Temporary: For Reviewer purpose only]
+
+`docker-compose up -d && robot --variable testType:docker --suite offers . && docker-compose down` - Run all test suites on self-hosted docker, change the suite name accordingly
+
+## To generate Reports using Allure [Temporary: For Reviewer purpose only]
+
+`pip install allure-robotframework`
+
+Add  `--listener 'allure_robotframework;./results/allure'` to any test command above.
+
+Example:
+
+`robot --listener 'allure_robotframework;./results/allure' --variable testType:single --suite offers .`
+
+ -->
+<!-- # Actual README [WIP] -->
```

Comment:
remove all temp stuff and update the readme such that the external user can follow and run the tests.